index.js

```diff
@@ -14,63 +14,81 @@ const symbolicate = async (options) => {
   const {force, file} = options
   const cacheDirectory = path.join(__dirname, 'cache', 'breakpad_symbols')
 
-  const symbolCache = new Map
   const dumpText = await fs.promises.readFile(file, 'utf8')
   const images = binaryImages(dumpText)
   const electronImage = images.find(v => /electron/i.test(v.library))
   if (electronImage) console.error(`Found Electron ${electronImage.version}`)
   else console.error('No Electron image found')
 
-  let result = []
+  const lines = dumpText.split(/\r?\n/);
```

Comment:
I've decided to just modify this array in-place instead of rebuilding it from scratch. It is faster, but more importantly I could do a bunch of early `continue`s in the loop and the code is not as deep anymore.